### PR TITLE
fix(appeals): fix problem with appeals assigned to me (a2-1828)

### PR DESCRIPTION
**Pull request by BenSurgisonPINS**
## Describe your changes

API:
 - Fixed statuses format
   
TESTING:
- API unit tests pass
- Manually tested within browser

## Issue ticket number and link

[A2-1828 Problem with Appeals assigned to me - BO](https://pins-ds.atlassian.net/browse/A2-1828)

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/api/src/server/endpoints/appeals/appeals.controller.js [hunk 1]
```diff
@@ -112,10 +112,15 @@ const getMyAppeals = async (req, res) => {
 		);
 		const sortedAppeals = sortAppeals(formattedAppeals);
 
+		// Flatten to an array of strings
```
Comment (james-hill-kc) [inline appeals/api/src/server/endpoints/appeals/appeals.controller.js:115]: The comment is probably not necessary, but it doesn't hurt

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Review by james-hill-kc — APPROVED** [review]
LGTM! Are any web app changes needed to adjust for the new statuses format?

File: appeals/api/src/server/endpoints/appeals/appeals.controller.js (hunk 1 above)
Comment (BenSurgisonPINS) [reply appeals/api/src/server/endpoints/appeals/appeals.controller.js:115]: No tests fail so it doesn't look like it.  I've also move around a few pages and it looks ok.

**Pull request merged**